Makefile

```diff
@@ -22,6 +22,9 @@ generate-grpc-files: configure-grpc-generator prepare-grpc-module tidy
 		--proto_path=$($@_COMMON_PROTO_DIR) \
 		$(shell find $($@_COMMON_PROTO_DIR) -name '*.proto' )
 
+check-grpc-files: generate-grpc-files
+	@git diff --name-only --exit-code ./*.go || { echo "Committed Protobuf files do not match the newly generated. Please, regenerate Protobuf and commit changes"; exit 1; }
+
```

Comment:
note: does not look like this works - the workflow has failed. Could you please take a look?

Reply:
Workflow failed because protoc prints version into generated file.
locally: 
```
// Code generated by protoc-gen-go. DO NOT EDIT.
// versions:
// 	protoc-gen-go v1.36.5
// 	protoc        v3.12.4
// source: th2_grpc_common/common.proto
```
github runner
protoc version = 3.6.1
https://github.com/th2-net/th2-grpc-common-go/actions/runs/13262344097/job/37021592612?pr=1#step:4:85

Reply:
We probably should install the right version on the runner

Reply:
I'm not sure that is correct way, because I can't control the protoc version on my local workstation.

Reply:
There are options to install protoc without admin rights
https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os